src/components/DND/DragDropContext.tsx

```diff
@@ -0,0 +1,145 @@
+import React from "react";
+import { Draggable } from "components/DND/Draggable";
+import Droppable from "components/DND/Droppable";
+
+const findScrollParents = (
+  element: HTMLElement | null
+): Array<{ vertical: boolean; element: HTMLElement }> => {
+  if (!element) return [];
+  let result = findScrollParents(element.parentElement);
+  if (element.scrollHeight > element.offsetHeight)
+    result = [{ vertical: true, element }, ...result];
+  if (element.scrollWidth > element.offsetWidth) result = [{ vertical: false, element }, ...result];
+  return result;
+};
```

Comment:
Can you make it so that when you drop a card in a column that has a scrollbar, that the column doesn't start back at the top of the column after the card has been dropped?

Reply:
This is actually a bug in master too, it's caused by the page load and fixed by the move endpoint change.